File: mifosng-android/src/main/java/com/mifos/api/local/MifosDatabase.java
package com.mifos.api.local;

import com.raizlabs.android.dbflow.annotation.Database;

/**
 * Created by Rajan Maurya on 23/06/16.
 */
@Database(name = MifosDatabase.NAME, version = MifosDatabase.VERSION, foreignKeysSupported = true)
public class MifosDatabase {

    // database name will be Mifos.db
    public static final String NAME = "Mifos";

    //Always Increase the Version Number
    public static final int VERSION = 2;
}

File: core/datastore/src/main/java/com/mifos/core/database/MifosDatabase.kt
package com.mifos.core.database

import com.raizlabs.android.dbflow.annotation.Database

/**
 * Created by Rajan Maurya on 23/06/16.
 */
@Database(name = MifosDatabase.NAME, version = MifosDatabase.VERSION, foreignKeysSupported = true)
object MifosDatabase {
    // database name will be Mifos.db
    const val NAME = "Mifos"

    //Always Increase the Version Number
    const val VERSION = 2
}